learn/advanced/chronological_filtering_sorting.md

```diff
@@ -0,0 +1,109 @@
+# Filtering and sorting with chronological values
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
+
+To filter and sort search results based on chronological values, your documents must have a numeric field containing a UNIX timestamp. In this videogames dataset, the release year is formatted as a timestamp:
+
+```json
+[
+  {
+    "id": 0,
+    "title": "Return of the Obra Dinn",
+    "genre": "adventure",
+    "release_date": 1514761200
+  },
+  {
+    "id": 1,
+    "title": "The Excavation of Hob's Barrow",
+    "genre": "adventure",
+    "release_date": 1640991600
+  },
+  {
+    "id": 2,
+    "title": "Bayonetta 2",
+    "genre": "action",
+    "release_date": 1388530800
+  }
+]
+```
+
+If your chronological data is expressed in datetime formats such as ISO 8601 like `"2022-01-01"`, you must convert it to a numeric value before indexing it with Meilisearch.
+
+Most programming languages have built-in tools to help you with this process:
+
+```js
+let game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "year": 2018
+};
+
+const timestamp = Date.parse(game.year);
+
+game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "year": 2018,
+  "release_date": timestamp
+};
+```
+
+:::tip
+When preparing your dataset, it can be useful to leave the date and time fields in your documents intact. In the example above, we keep the `year` field because it is more readable than the raw `timestamp`.
+:::
+
+After adding a numeric timestamp to all documents, index your dataset as usual.
+
+## Chronological filtering
```

Comment:
I have chosen to use "Chronological" throughout the article to avoid entering the pitfall of date (YYYY-MM-DD) vs time (18:27).

Is "chronological" too complex a word?

Reply:
You can ignore this if you want
Doesn't chronological mean arranging in the order it happened? Chronological sorting makes sense to me; you're sorting by oldest to newest or the other way round, but not chronological filtering. I didn't find any search results for `chronological filtering`. Should we go with something like Filtering or sorting by date? It will also get rid of any complexity "chronological" brings

Reply:
Doesn't filtering by date, generally, mean you can filter by time too? At least, that's how I see it